src/polus/plugins/_plugins/classes/plugin_classes.py

```diff
@@ -38,90 +36,49 @@
 Paths and Fields
 """
 # Location to store any discovered plugin manifests
-PLUGIN_DIR = pathlib.Path(__file__).parent.parent.joinpath("manifests")
-
-
-def load_config(config: typing.Union[dict, pathlib.Path]):
-    """Load configured plugin from config file/dict."""
-    if isinstance(config, pathlib.Path):
-        with open(config) as fr:
-            m = json.load(fr)
-    elif isinstance(config, dict):
-        m = config
-    else:
-        raise TypeError("config must be a dict or a path")
-    _io = m["_io_keys"]
-    cl = m["class"]
-    m.pop("class", None)
-    if cl == "Compute":
-        pl = ComputePlugin(_uuid=False, **m)
-    elif cl == "WIPP":
-        pl = Plugin(_uuid=False, **m)
-    else:
-        raise ValueError("Invalid value of class")
-    for k, v in _io.items():
-        val = v["value"]
-        if val is not None:  # exclude those values not set
-            setattr(pl, k, val)
-    return pl
-
-
-def get_plugin(name: str, version: typing.Optional[str] = None):
-    """Get a plugin with option to specify version.
-
-    Return a plugin object with the option to specify a version. The specified version's manifest must exist in manifests folder.
-
-    Args:
-        name: Name of the plugin.
-        version: Optional version of the plugin, must follow semver.
-
-    Returns:
-        Plugin object
-    """
-    if version is None:
-        return load_plugin(PLUGINS[name][max(PLUGINS[name])])
-    else:
-        return load_plugin(PLUGINS[name][Version(**{"version": version})])
+_PLUGIN_DIR = Path(__file__).parent.parent.joinpath("manifests")
 
 
-def refresh():
+def refresh() -> None:
     """Refresh the plugin list."""
     organizations = [
-        x for x in PLUGIN_DIR.iterdir() if x.name != "__pycache__"
+        x for x in _PLUGIN_DIR.iterdir() if x.name != "__pycache__" and x.is_dir()
     ]  # ignore __pycache__
```

Comment:
All plugin folder names end in `-plugin`. Wouldn't that be more useful?

Reply:
These are different. Names of directories are the names of organizations (labshare, polusai...)